demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -2695,20 +2758,21 @@ def add_ignored_errors_to_list(self, config, section, key, ignored_errors_list):
                 self.create_ignored_errors_list(PRESET_ERROR_TO_CHECK.get(key))
             )
 
-    def get_error_ignore_list(self, pack_name):
+    def get_error_ignore_list(self, pack_name="", config=None):
         ignored_errors_list: dict = {}
         if pack_name:
-            if config := get_pack_ignore_content(pack_name):
-                # create file specific ignored errors list
-                for section in filter(
-                    lambda section: section.startswith("file:"), config.sections()
-                ):
-                    file_name = section[5:]
-                    ignored_errors_list[file_name] = []
-                    for key in config[section]:
-                        self.add_ignored_errors_to_list(
-                            config, section, key, ignored_errors_list[file_name]
-                        )
+            config = get_pack_ignore_content(pack_name)
+        if config:
+            # create file specific ignored errors list
+            for section in filter(
+                lambda section: section.startswith("file:"), config.sections()
+            ):
+                file_name = section[5:]
```

Comment:
whats 5?

Reply:
lets not use magic numbers wherever possible

Reply:
```suggestion
                file_name = section[len("files"):]
```